rules/windows/process_creation/process_creation_win_manipulation_of_tokens_via_impersonate_binary.yml

```diff
@@ -0,0 +1,30 @@
+title: Impersonate Execution
+id: cf0c254b-22f1-4b2b-8221-e137b3c0af94
+status: experimental
+description: Detects execution of the Impersonate tool. Which can be used to manipulate tokens on a Windows computers remotely (PsExec/WmiExec) or interactively
+references:
+    - https://sensepost.com/blog/2022/abusing-windows-tokens-to-compromise-active-directory-without-touching-lsass/
+    - https://github.com/sensepost/impersonate
+author: Sai Prashanth Pulisetti @pulisettis
+date: 2022/12/21
+tags:
+    - attack.privilege_escalation
+    - attack.defense_evasion
+    - attack.t1134.001
+    - attack.t1134.003
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        HostApplication|contains|all:
+            - 'impersonate.exe'
+        HostApplication|contains:
```

Comment:
The rule is still using the `HostApplication`. Can you apply the @frack113 suggestion

Reply:
Done, made changes accordingly.

Reply:
Where are the changes? I don't see the selections and the hashes?